controllers/actions.github.com/resourcebuilder.go

```diff
@@ -133,7 +130,10 @@ func (b *resourceBuilder) newScaleSetListenerPod(autoscalingListener *v1alpha1.A
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      autoscalingListener.Name,
 			Namespace: autoscalingListener.Namespace,
-			Labels:    newLabels,
+			Labels: map[string]string{
+				"auto-scaling-runner-set-namespace": autoscalingListener.Spec.AutoscalingRunnerSetNamespace,
```

Comment:
Do we want to include listener in the label name?

Reply:
Not sure, because the listener name is the name of the pod, you can easily search for that pod without passing the `-l`